final_task/rss_reader/rss_reader.py

```diff
@@ -125,6 +134,8 @@ def main():
     except requests.exceptions.ConnectionError:
         logging.critical("Sorry, you have an proxy or SSL error")
         # A proxy or SSL error occurred.
+    except pisa:
```

Comment:
Check what exactly `pisa` object is. I doubt it it an Exception object

Reply:
Sorry I overlooked it, will fix